pkg/arvo/app/group-store.hoon

```diff
@@ -1,26 +1,62 @@
-::  group-store: data store for groups of ships
+::  group-store: Store groups of ships
 ::
-/-  *group-store
-/+  default-agent, verb, dbug
+::    group-store stores groups of ships, so that resources in other apps can be
+::    associated with a group. The current model of group-store rolls
+::    permissions and invites inside this store for simplicity reasons, although
+::    these should be prised apart in a future revision of group store.
+::
+::
+::    ## Scry paths
+::
+::    /y/groups:
+::      A listing of the current groups
+::    /y/groups/[group-id]/tag-queries:
+::      A listing of the tag queries for a group
+::    /x/groups/[group-id]:
+::      The group itself
```

Comment:
No subsequent `%x` for "ships with tag"? (:

(The "tag query" phrasing feels pretty ugly to me (here and everywhere), too, so I'd probably do something like this:)

```hoon
/y/groups/[group-id]/tags        ::  get tags used in group
/x/groups/[group-id]/tags/[tag]  ::  get ships by tag
```

Reply:
Considering nixing the `tag-query` scry tbh because we can just implement it as a scry helper in `lib/group`. Are subsequent scries in the same invocation cached? Maybe could help performance.